### PR TITLE
fix(List): Added aria roles for list/list-item

### PR DESCRIPTION
**Pull request by fracmak**
List elements are missing the appropriate roles to indicate what is being displayed.

Documentation: https://www.w3.org/TR/wai-aria/roles#list

### CODE REVIEW

File: test/specs/elements/List/List-test.js [hunk 1]
```diff
@@ -37,12 +37,14 @@ describe('List', () => {
       const wrapper = shallow(<List />)
 
       wrapper.find('ListItem').should.have.lengthOf(0)
+      wrapper.should.have.prop('role', 'list')
```
Comment (levithomason) [inline test/specs/elements/List/List-test.js]: Would prefer we add separate `it()` blocks for the accessibility assertions:

```jsx
it('is accessible', () => {...})
// or 
it('has proper aria props', () => {...})
```

Something to this effect.  The idea is to keep the `it()` strings descriptive to exactly what is being asserted inside of them.

**Review by levithomason — CHANGES_REQUESTED** [review]
Just one comment regarding organizing tests.  Looks great otherwise 👍

**Comment by levithomason** [comment]
The test organization used in https://github.com/Semantic-Org/Semantic-UI-React/pull/1006 is exactly what I was thinking.  Let's do that here as well.

**Comment by fracmak** [comment]
tests reorganized

**Comment by levithomason** [comment]
Made a trivial fix here, mostly to debug why CI is suddenly not running at all... Hm.

**Comment by levithomason** [comment]
So odd, gonna have to double back and test these manually.

**Comment by levithomason** [comment]
Releasing now as `semantic-ui-react@0.62.1`.

**Pull request merged**